config.md

```diff
@@ -90,7 +90,9 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 
 ## Process configuration
 
-* **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`terminal`** (bool, optional) specifies whether a terminal is attached to that process, defaults to false.
+  On Linux, a psuedo-terminal pair is allocated for the container process and the standard streams are duped to the pty slave.
```

Comment:
For extra clarity (and to match “Atomically duplicate 'oldfd' on 'newfd'” wording in [dup(2)][1] and “pseudoterminal” in [pts(4)][2]), I'd recommend:

> On Linux, a psuedoterminal pair is allocated for the container process and the psuedoterminal slave is duplicated on the container process's [standard streams][stdin.3].
>
> [stdin.3]: http://man7.org/linux/man-pages/man3/stdin.3.html

[1]: http://man7.org/linux/man-pages/man2/dup.2.html#NOTES
[2]: http://man7.org/linux/man-pages/man4/pts.4.html